framework/components/ecl-forms/ecl-forms-text-inputs/ecl-forms-text-inputs.twig

```diff
@@ -1,29 +1,21 @@
 <form>
-  <div class="form-group">
+  <div class="ecl-form-group">
     <label for="example-input-id-1">Text Input - default with placeholder</label>
     <p class="help-block">This is some help text.</p>
-    <input type="text" class="form-control" id="example-input-id-1" placeholder="Some placeholder text."/>
+    <input type="text" class="ecl-text-input" id="example-input-id-1" placeholder="Some placeholder text."/>
```

Comment:
No missing examples actually: they made an example for the `:focus` state by adding a class for that... Do we want to follow this road?